web/src/components/storage/iscsi/InitiatorSection.tsx

```diff
@@ -24,17 +24,46 @@ import React from "react";
 
 import { _ } from "~/i18n";
 import { Page } from "~/components/core";
-import { InitiatorPresenter } from "~/components/storage/iscsi";
-import { useInitiator, useInitiatorChanges } from "~/queries/storage/iscsi";
+import {
+  DescriptionList,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  DescriptionListDescription,
+} from "@patternfly/react-core";
+import { InitiatorForm } from "~/components/storage/iscsi";
+import { useInitiator, useInitiatorMutation, useInitiatorChanges } from "~/queries/storage/iscsi";
+
+const InitiatorDescription = ({ initiator }) => {
+  return (
+    <DescriptionList aria-label={_("Initiator details")} isHorizontal>
+      <DescriptionListGroup>
+        <DescriptionListTerm>{_("Name")}</DescriptionListTerm>
+        <DescriptionListDescription>{initiator.name}</DescriptionListDescription>
+      </DescriptionListGroup>
+    </DescriptionList>
+  );
+};
 
 export default function InitiatorSection() {
   const initiator = useInitiator();
+  const { mutateAsync: updateInitiator } = useInitiatorMutation();
   useInitiatorChanges();
 
+  const submitForm = async ({ name }) => {
+    await updateInitiator({ name });
+  };
+
+  const desc = initiator.ibft
+    ? _("Configuration read from the iSCSI Boot Firmware Table (iBFT).")
+    : _(
+        "No iBFT (iSCSI Boot Firmware) configuration found. The initiator can be configured manually.",
```

Comment:
NP: I'd keep the naming equal/stable in both branches. I.e.,

"No **iSCSI Boot Firmware Table (iBFT)** found. The initiator name can be configured manually"